views/update/update-layout.html

```diff
@@ -0,0 +1,95 @@
+{% extends "govuk/template.njk" %}
```

Comment:
duplication with "layout.html" here but I think it's ok to duplicate in order to keep it simpler to diverge.

Reply:
yeah , tried avoiding it but its harder to do without it , else it might impact the Reg journey